packages/web/app/src/lib/preflight/shared-types.ts

```diff
@@ -2,6 +2,14 @@
 
 import { Kit } from '../kit';
 
+// todo stop using enums
```

Comment:
why are enums bad?

Reply:
Its non-standard ES syntax and less type safe than unions in various situations.

Its a pretty well documented topic so maybe have a search about it, or even ask AI. If you find reasons to still use them, happy to discuss.

Reply:
Per @n1ru4l's request this will be moved to another PR for discussion.